openmc/mgxs_library.py

```diff
@@ -1163,52 +1379,35 @@ def remove_xsdata(self, xsdata):
 
         self._xsdatas.remove(xsdata)
 
-    def _create_groups_subelement(self):
-        if self._energy_groups is not None:
-            element = ET.SubElement(self._cross_sections_file, 'groups')
-            element.text = str(self._energy_groups.num_groups)
-
-    def _create_group_structure_subelement(self):
-        if self._energy_groups is not None:
-            element = ET.SubElement(self._cross_sections_file,
-                                    'group_structure')
-            element.text = ' '.join(map(str, self._energy_groups.group_edges))
-
-    def _create_inverse_velocities_subelement(self):
-        if self._inverse_velocities is not None:
-            element = ET.SubElement(self._cross_sections_file,
-                                    'inverse_velocities')
-            element.text = ' '.join(map(str, self._inverse_velocities))
-
-    def _create_xsdata_subelements(self):
-        for xsdata in self._xsdatas:
-            xml_element = xsdata._get_xsdata_xml()
-            self._cross_sections_file.append(xml_element)
-
-    def export_to_xml(self, filename='mgxs.xml'):
-        """Create an mgxs.xml file that can be used for a
-        simulation.
+    def export_to_hdf5(self, filename='mgxs.h5', compression=None):
+        """Create an mgxs.h5 file that can be used for a simulation.
 
         Parameters
         ----------
-        filename : str, optional
-            filename of file, default is mgxs.xml
+        filename : str
+            Filename of file, default is mgxs.xml.
+        compression : None or int
+            The compression level to use in the datasets within the hdf5 file.
+            A value of None implies no compression, numbers between 0 and 9
+            refer to the compression level using the gzip algorithm.
+            Defaults to None.
```

Comment:
Is having the compression level option necessary? I feel like this will never get used.